packages/rollup-core/test/app/canonical-chain-batch-submitter.spec.ts

```diff
@@ -275,6 +275,7 @@ describe('Canonical Chain Batch Submitter', () => {
           calldata: keccak256FromUtf8('some calldata'),
           stateRoot: keccak256FromUtf8('l2 state root'),
           signature: 'ab'.repeat(65),
+          type: 0,
```

Comment:
Do we know what these types will be? Either way, might make sense to create an enum / set of constants so we have some validation and we're not dealing with random numbers.

Reply:
Good call, the types are defined in this solidity contract.

https://github.com/ethereum-optimism/optimism-monorepo/pull/243/files#diff-baf9b9c3d2f365ed42b389af41d9e5d3R21-R25

This definitely needs to be defined in an enum